src/components/interfaces/MOBILE_API.xml

```diff
@@ -5369,65 +5454,149 @@
 
   <function name="OnEncodedSyncPData" functionID="OnEncodedSyncPDataID" messagetype="notification" >
     <description>
-    	Callback including encoded data of any SyncP packets that SYNC needs to send back to the mobile device.
-    	Legacy / v1 Protocol implementation; responds to EncodedSyncPData.
-    	*** DEPRECATED ***
+      Callback including encoded data of any SyncP packets that SYNC needs to send back to the mobile device.
+      Legacy / v1 Protocol implementation; responds to EncodedSyncPData.
+      *** DEPRECATED ***
     </description>
     <param name="data" type="String" maxlength="1000000" minsize="1" maxsize="100" mandatory="true" array="true">
       <description>Contains base64 encoded string of SyncP packets.</description>
     </param>
     <param name="URL" type="String" maxlength="1000" mandatory="false">
       <description>
-      	If blank, the SyncP data shall be forwarded to the app.
-      	If not blank, the SyncP data shall be forwarded to the provided URL.
+        If blank, the SyncP data shall be forwarded to the app.
+        If not blank, the SyncP data shall be forwarded to the provided URL.
       </description>
     </param>
     <param name="Timeout" type="Integer" minvalue="0" maxvalue="2000000000" mandatory="false">
       <description>
-      	If blank, the SyncP data shall be forwarded to the app.
-      	If not blank, the SyncP data shall be forwarded with the provided timeout in seconds.
+        If blank, the SyncP data shall be forwarded to the app.
+        If not blank, the SyncP data shall be forwarded with the provided timeout in seconds.
       </description>
     </param>
   </function>
 
-<function name="SetAudioStreamingIndicator" functionID="SetAudioStreamingIndicatorID" messagetype="request">
-   <description>Sets the icon properly to the current audio playback of the app.</description>
-   <param name="audioStreamingIndicator" type="AudioStreamingIndicator" mandatory="true">
-    <description>
-     See AudioStreamingIndicator.
-    </description>
-   </param>
-</function>
+  <function name="SubscribeWayPoints" functionID="SubscribeWayPointsID" messagetype="request">

```

Comment:
@LuxoftAKutsan reveted in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215?w=1